extensions/amp-analytics/0.1/vendors.js

```diff
@@ -89,14 +89,23 @@ export const ANALYTICS_CONFIG = {
         'g3=${sponsorName}&' +
         'g4=${contentType}&' +
         'x=${scrollTop}&' +
+        'y=${scrollHeight}&' +
         'w=${screenHeight}&' +
         'j=${decayTime}&' +
         'r=${documentReferrer}&' +
-        't=${clientId(_cb_amp)}${pageViewId}&' +
-        'i=${title}',
+        'b=${pageLoadTime}&' +
+        't=${clientId(_cb)}${pageViewId}&' +
+        'i=${title}&' +
+        'T=${timestamp}&' +
+        'E=${totalEngagedTime}&' +
+        'C=2&' +
+        'R=1&' +
+        'W=0&' +
+        'I=0&' +
+        'c=120',
       'baseSuffix': '&_',
-      'interval': '${host}${basePrefix}&${baseSuffix}',
-      'anchorClick': '${host}${basePrefix}&${baseSuffix}'
+      'interval': '${host}${basePrefix}${baseSuffix}',
+      'anchorClick': '${host}${basePrefix}${baseSuffix}'
```

Comment:
please add trailing comma. It is apparently valid in javascript now and reduces diffs :).